src/pkg/cli/composeUp.go

```diff
@@ -30,30 +31,37 @@ func buildContext(force bool) compose.BuildContext {
 }
 
 // ComposeUp validates a compose project and uploads the services using the client
-func ComposeUp(ctx context.Context, c client.Client, force bool) (*defangv1.DeployResponse, *types.Project, error) {
+func ComposeUp(ctx context.Context, c client.Client, force bool) (*defangv1.DeployResponse, *types.Project, bool, error) {
+	bypassSubscription := false
```

Comment:
There's a bit of a mismatch here, between the responsibility of `ComposeUp` and the idea of subscriptions. This function shouldn't have to know about whether or not the caller intends to subscribe. Not sure if I'm explaining this well. Subscribe is something higher up, a CLI concept, so it feels out of place inside of ComposeUp.